### DISCUSSION TITLE
What's the pronunciation of 'zustand'?

### DISCUSSION THREAD

**Discussion by ZeroCho** [pmndrs/zustand#1032] [Q&A]
Sorry for meaningless issue,
But I'm just curious what you(creator) intended when you made this.
English style pronunciation or German style? Cuz they are very different.

I usually call a name of library by english style but as I noticed that 'zustand' is real german word, I'm little bit confused. :)

**Comment by dai-shi** [comment]
As far as I remember, the author is unopinionated about how it should be read, either in German style, English style or even others.

**Comment by vladimir-light** [comment]
Maybe include pronunciation/phonetic transcription to a README.md ?

Something like:
**Zustand** _[ˈtsuːʃtant]_ - a german word for state.

**Reply by drcmda** [reply]
i've had fun seeing people pronounce it "wrong" all over youtube and at conferences, and i really don't mind, even happy they're finding pronounciations that aren't so hard on the tongue. i think i'd prefer to be not so pointy about it. zoostand, soushtand, tsutand, it's all good.

**Comment by OlafHaag** [comment]
The bear in the illustration of readme/docs gets it wrong, as it separates the word into "zus" & "tand". That may contribute to the use of "alternative" pronounciations of the word. As a native German speaker, that immediately caught my attention. 😅
When you separate the word Zustand into 2 lines, it should be:

> Zu-
> stand

**Reply by dai-shi** [reply]
btw, I read zus-tand as English (?) pronunciation.
